Python/ceval.c

```diff
@@ -699,12 +699,6 @@ PyEval_EvalFrameEx(PyFrameObject *f, int throwflag)
 #define PRE_DISPATCH_GOTO() ((void)0)
 #endif
 
-#define NOTRACE_DISPATCH() \
-    { \
-        NEXTOPARG(); \
-        PRE_DISPATCH_GOTO(); \
-        DISPATCH_GOTO(); \
-    }
```

Comment:
Should we re-run the benchmarks to make sure this doesn't slow things down? It's just one instruction, `opcode |= cframe.use_tracing OR_DTRACE_LINE;` -- but still, `cframe.use_tracing` seems to be a forced memory load.

Reply:
Never mind, this was covered in the issue. No need to do this again.